gui/dialogs/aircraft-dialog.xml

```diff
@@ -125,11 +125,33 @@
                 <property>/engines/active-engine/oil_consumption_allowed</property>
                 <live>true</live>
                 <binding>
+                    <condition>
+                        <equals>
+                            <property>/controls/engines/active-engine</property>
+                            <value type="int">0</value>
+                        </equals>
+                    </condition>
                     <command>property-assign</command>
                     <property>/engines/active-engine/oil-level</property>
                     <value>7.0</value>
```

Comment:
@dany93 since you already removed the binding that was setting the oil to 7 quarts when toggling the carb icing feature, you could as well remove this binding from the oil consumption toggle and so we solve everything with a single PR, what do you think? I am referring to the 3 lines above my comment